BSP/Simulator/DataGeneration/ltc6811.py

```diff
@@ -0,0 +1,647 @@
+    # LTC6811 objects
+    minions = []
+    voltages = [25000, 26000, 27000, 28000, 39000, 40000, 41000, 42000, 0, 0, 0, 0]
+    temperatures = [[25000, 25001], [26000, 26001], [27000, 27001], [28000, 28001], [29000, 29001], [30000, 30001], [31000, 31001], [32000, 32001]]
+    config = [0, 1, 2, 3, 4, 5]
+    for i in range(4):
+        minions.append(LTC6811(None))
+        minions[i].set_voltage(voltages)
+        minions[i].set_temperatures(temperatures)
+
+    # Unit Testing
+
+    # Test xxcfga_handler
+    fail = 0
+    minions[0].rx_data = config
+    wrcfga_handler(minions[0])
+    if minions[0].config_reg != config:
+        fail = 1
+
+    fail = 0
+    rdcfga_handler(minions[0])
+    if minions[0].tx_data != config:
+        fail = 1
+
+    # Test rdcva_handler
+    rdcva_handler(minions[0])
+    tx_idx = 0
+    fail = 0
+    for i in range(3):
+        batt_voltage = voltages[cv_groups['A']+i]
+        tx_voltage = (minions[0].tx_data[tx_idx] & 0x00FF) | ((minions[0].tx_data[tx_idx+1] & 0x00FF) << 8)
+        tx_idx += 2
+        if batt_voltage != tx_voltage:
+            fail = 1
+
+    # Test rdcvb_handler
+    rdcvb_handler(minions[0])
+    tx_idx = 0
+    fail = 0
+    for i in range(3):
+        batt_voltage = voltages[cv_groups['B']+i]
+        tx_voltage = (minions[0].tx_data[tx_idx] & 0x00FF) | ((minions[0].tx_data[tx_idx+1] & 0x00FF) << 8)
+        tx_idx += 2
+        if batt_voltage != tx_voltage:
+            fail = 1
+
+    # Test rdcvc_handler
+    rdcvc_handler(minions[0])
+    tx_idx = 0
+    fail = 0
+    for i in range(3):
+        batt_voltage = voltages[cv_groups['C']+i]
+        tx_voltage = (minions[0].tx_data[tx_idx] & 0x00FF) | ((minions[0].tx_data[tx_idx+1] & 0x00FF) << 8)
+        tx_idx += 2
+        if batt_voltage != tx_voltage:
+            fail = 1
+
+    # Test ADOWPU
+    minions[0].set_single_wire(2, 1)
+    adowpu_md0_dcp0_ch0_handler(minions[0])
+    rdcva_handler(minions[0])
+    tx_idx = 0
+    fail = 0
+    batt_voltage = [30000, 30000, 40000]
+    for i in range(3):
+        tx_voltage = (minions[0].tx_data[tx_idx] & 0x00FF) | ((minions[0].tx_data[tx_idx+1] & 0x00FF) << 8)
+        tx_idx += 2
+        if batt_voltage[i] != tx_voltage:
+            fail = 1
+
+    # Test ADOWPD
+    minions[0].set_single_wire(2, 1)
+    adowpd_md0_dcp0_ch0_handler(minions[0])
+    rdcva_handler(minions[0])
+    tx_idx = 0
+    fail = 0
+    batt_voltage = [40000, 40000, 30000]
+    for i in range(3):
+        tx_voltage = (minions[0].tx_data[tx_idx] & 0x00FF) | ((minions[0].tx_data[tx_idx+1] & 0x00FF) << 8)
+        tx_idx += 2
+        if batt_voltage[i] != tx_voltage:
+            fail = 1
+
+    # Test WRCOMM and STCOMM
+    minions[0].rx_data = [0xF9, 0x0F, 0xFF, 0x0F, 0xFF, 0xFF]
+    wrcomm_handler(minions[0])
+    fail = 0
+    if minions[0].ltc1380s[mux_idxs['SIM_LTC1380_MUX1']].mux_addr != mux_addresses['SIM_LTC1380_MUX1'] or minions[0].ltc1380s[mux_idxs['SIM_LTC1380_MUX1']].mux_en is True:
```

Comment:
A little nicer to read:
```suggestion
    if minions[0].ltc1380s[mux_idxs['SIM_LTC1380_MUX1']].mux_addr != mux_addresses['SIM_LTC1380_MUX1'] or \
       minions[0].ltc1380s[mux_idxs['SIM_LTC1380_MUX1']].mux_en is True:
```